### ISSUE TITLE
Allow writing a copy of an open Realm to a new file

### ISSUE THREAD

**Issue by bdash** [realm/realm-browser-osx#55]
When determining whether excessive file sizes is caused by version pinning a helpful first step can be to write a copy of the Realm to a new file to see whether that shrinks the file. It'd be convenient if the browser supported doing this.

**Comment by TimOliver** [comment]
@bdash Sure thing! Would a simple 'Export Copy...' option in the File menu suffice for that?

**Comment by segiddins** [comment]
Maybe `Save As...` ?

**Comment by bmunkholm** [comment]
"Compress File As"?

On Wed, Aug 19, 2015 at 6:56 PM, Samuel E. Giddins <notifications@github.com

**Comment by astigsen** [comment]
> "Compress File As"?

I would just say "Save As...". The fact that the file can get smaller is
just a happy side effect (but we might consider including an `optimize()`
to make it even better :-)).

Saying "Compress File As"? could easily lead people to believe that the
resulting file was zipped or similar.

**Comment by TimOliver** [comment]
Done! I've created a WIP PR here: https://github.com/realm/realm-browser-osx/pull/56; let me know what you think. :)

I've tentatively gone with "Save As..." for the menu item text, but I think we should reconsider this. I don't think we should use any terminology that implies that Realm files need to be manually saved. I chose "Export Copy..." since it was aimed to explicitly denote that a static copy of this Realm file is being made and being saved to disk. :)
